js/ui/popover.js

```diff
@@ -67,11 +68,18 @@ const getEventDelay = function(that, optionName) {
     return isObject(optionValue) && optionValue.delay;
 };
 const attachEvent = function(that, name) {
-    const target = that.option('target');
+    const { target, shading, disabled } = that.option();
     const isSelector = isString(target);
-    const event = getEventName(that, name + 'Event');
+    let event = getEventName(that, `${name}Event`);
+    const isClickEvent = event === 'click' || event === 'dxclick';
```

Comment:
How does it work?
Interop with the `target` element will be blocked by the shader (wrapper) element.
Popover will close on external click thanks to the `closeOnOutsideClick` option.
I think we could remove this const.

Reply:
Yes, but this help to avoid situation, when user set `shaiding` as `true`, and `hideEvent` as `click`. Without this check, he'll also get warning and that seems wrong

Reply:
Done